src/filter/defish0r/defish0r.c

```diff
@@ -37,6 +37,39 @@
 
 double PI=3.14159265358979;
 
+//add simplified 'Elestic Scale' to fix superview
+float stretchWidth(int width, int widthCentre, float currentXPos, float stretchFactor)
+{
+	double ratio, lowerWeight = 0, higherWeight = 0, linearRatio;
+	unsigned int lengthSection;
+	float relativeXPos = 0;
+
+	//midline?
+	if (currentXPos < (float)widthCentre)
+	{
+		lengthSection = widthCentre - 1;
+		linearRatio = (double) ( currentXPos ) / lengthSection;
+		ratio = sin(linearRatio * PI - PI) * stretchFactor + linearRatio;
+	}
+	else
+	{
+		lengthSection = width - widthCentre - 1;
+		linearRatio = (double) ( currentXPos - widthCentre ) / lengthSection;
+		ratio = sin(linearRatio * PI) * stretchFactor + linearRatio;
+	}
+	ratio = ratio <= 0 ? 0 : ratio;
```

Comment:
Add `.0` suffix to these numeric literals to make them match the type of `ratio`.